Makefile

```diff
@@ -73,7 +73,7 @@ GOARCH ?= amd64
 
 LDFLAGS := $(shell cat hack/make/ldflags.txt)
 LDFLAGS_CSI := $(LDFLAGS) -X "$(MOD_NAME)/pkg/csi/service.version=$(VERSION)"
-LDFLAGS_SYNCER := $(LDFLAGS)
+LDFLAGS_SYNCER := $(LDFLAGS) -X "$(MOD_NAME)/pkg/csi/service.version=$(VERSION)"
```

Comment:
is `"$(MOD_NAME)/pkg/csi/service.version=$(VERSION)"` needed for syncer? The `service` package is only used by CSI controller, not syncer.

Reply:
reverted back to `LDFLAGS_SYNCER := $(LDFLAGS)`